src/main/java/secondmind/Parser.java

```diff
@@ -0,0 +1,39 @@
+package secondmind;
+
+public class Parser {
+    public Parser() {}
+
+    public String[] processInput(String input) throws NumberFormatException {
+        String[] newInput = input.split(" ");
+        String[] ignoreInstruction = new String[] {"ignore"};
```

Comment:
Perhaps the `newInput` and `ignoreInstruction` can be pluralised? Even though there's one item, it's still a list that can hold multiple items.